tests/unit/cmd/honeytoken/test_honeytoken.py

```diff
@@ -0,0 +1,62 @@
+from click.testing import CliRunner
+
+from ggshield.cmd.main import cli
+from ggshield.core.errors import ExitCode
+from tests.unit.conftest import my_vcr
+
+
+@my_vcr.use_cassette("test_honeytoken_create_no_argument")
+def test_honeytokens_create_no_arg(cli_fs_runner: CliRunner) -> None:
+    """
+    GIVEN -
+    WHEN running the honeytoken command with no argument
+    THEN the return code is 2
+    """
+
+    result = cli_fs_runner.invoke(
+        cli,
+        [
+            "honeytoken",
+            "create",
+        ],
+    )
+    assert result.exit_code == ExitCode.USAGE_ERROR
```

Comment:
We have 2 helper functions for these tests: `assert_invoke_ok()` and `assert_invoke_exited_with()`. The benefit of using them is they print the full stdout and stderr if the result is not what is expected. Can you make the tests use them instead of  `assert`?